mobile/assets/locales/ro/translations.json

```diff
@@ -101,21 +101,28 @@
       }
     },
     "actions": {
-      "next_step": "Next step",
-      "back": "Back",
-      "skip": "Skip this step",
-      "finalize": "Finalize"
+      "next_step": "Pasul urmator",
+      "back": "Inapoi",
+      "skip": "Sari peste acest pas",
+      "finalize": "Finalizeaza"
     }
   },
   "polling_station_information": {
     "header": {
       "title": "Informații despre secția de votare"
+    },
+    "actions": {
+      "submit": "Submiteti raspunsurile"
     }
   },
   "bottom_sheets": {
     "observations": {
+      "title": "Sectile mele de votare",
+      "paragraph": "Puteți comuta între secțiile de votare dacă doriți să revizuiți răspunsurile din formulare sau informațiile despre secțiile de votare.",
       "actions": {
-        "clear_form": "Ștergeți formularul (ștergeți toate răspunsurile)"
+        "clear_form": "Ștergeți formularul (ștergeți toate răspunsurile)",
+        "add_station": "Adăugați o nouă secție de votare.",
```

Comment:
Please remove the '.' from the end as this text is displayed inside a button

Reply:
done